src/bp/sdk/botpress.d.ts

```diff
@@ -866,6 +866,7 @@ declare module 'botpress/sdk' {
     author?: string
     disabled?: boolean
     private?: boolean
+    needsYourApproval?: boolean
```

Comment:
How come this is stored in the bot config ? really ?

Reply:
I just put it there for now, didn't know where it would be, but the bot will be the thing holding whether it needs to be approved or not, no?

Reply:
right but under the pipeline_status property I guess it would look like 
```
{
  /*
    ... other bot config fields
  */
  "pipeline_status": {
    "current_stage": {
      "id": "dev", // id of the current stage
      "promoted_on": "2019-04-03T01:08:46.999Z", // date the bot has moved to this stage
      "promoted_by": "user@botpress.com" //email of the user who changed the stage of the bot
    },
    "stage_request": {
      "id": "staging", //id of the desired stage
      "requested_on": "2019-04-04T13:16:32.107Z",
      "requested_by": "user2@botpress.com" //email of the user who asked for a stage change
      "approvals": ["reviewer@botpress.com", "reviewer2@botpress.comn"]
    }
  }
}

This way we can easily compare approvals vs required approvals

Reply:
ehhh I'm confused 😂 just tell me where you want it, I wasn't suggesting in any ways where or what it should be, I just wanted to put a placecholder